### PR TITLE
Jcsz1 fusy 6/11

### PR DESCRIPTION
**Pull request by prz3m3**

### CODE REVIEW

File: WalutyBusinessLogic/CurrencyConvertion/CurrencyConvertion.cs [hunk 1]
```diff
@@ -1,44 +1,36 @@
...
-        CurrencyForConversation secondCurrencyConvertion = new CurrencyForConversation();
+        private float ValueFirstCurrency { get; set; }
+        private float AmountFirstCurrency { get; set; }
+        private float ValueSecondCurrency { get; set; }
+        private float AmountSecondCurrency { get; set; }
+        
+        public float ConvertCurrency(float amountFirstCurrency, string firstNameCurrency,
+            string secondNameCurrency, int date)
+        {
+            AmountFirstCurrency = amountFirstCurrency;
```
[the remark this answers is not in the document]
Comment (prz3m3) [reply WalutyBusinessLogic/CurrencyConvertion/CurrencyConvertion.cs]: Ok, I remove this property.

File: WalutyBusinessLogic/CurrencyConvertion/CurrencyConvertion.cs [hunk 2]
```diff
@@ -1,44 +1,36 @@
...
 
 namespace WalutyBusinessLogic
 {
-    class CurrencyConvertion
+    public class CurrencyConvertion
     {
-        CurrencyRecord currencyRecord = new CurrencyRecord();
-        CurrencyForConversation firstCurrencyConvertion = new CurrencyForConversation();
-        CurrencyForConversation secondCurrencyConvertion = new CurrencyForConversation();
+        private float ValueFirstCurrency { get; set; }
```
[the remark this answers is not in the document]
Comment (prz3m3) [reply WalutyBusinessLogic/CurrencyConvertion/CurrencyConvertion.cs]: You're right, If I change properties to public , will be ok?

File: WalutyBusinessLogic/CurrencyConvertion/CurrencyConvertion.cs [hunk 3]
```diff
@@ -1,44 +1,32 @@
...
 
 namespace WalutyBusinessLogic
 {
-    class CurrencyConvertion
+    public class CurrencyConvertion
     {
-        CurrencyRecord currencyRecord = new CurrencyRecord();
-        CurrencyForConversation firstCurrencyConvertion = new CurrencyForConversation();
-        CurrencyForConversation secondCurrencyConvertion = new CurrencyForConversation();
+        public float ValueFirstCurrency { get; set; }
```
Comment (TDerkacz) [inline WalutyBusinessLogic/CurrencyConvertion/CurrencyConvertion.cs]: Properties are used to communicate with a class, not for internal operations only. If you want to use them, make a Constructor for class CurrencyConvertion so the usage of the class will be like this:
CurrencyConvertion("USD.txt", "EUR.txt"); 
That for sure will simplify utilization of the class.

**Comment by TDerkacz** [comment]
Where is UI for the functionalities? Please add them in new PR.

**Pull request merged**